### ISSUE TITLE
get workshop materials from intermine group and modify to suit our needs

### ISSUE THREAD

**Issue by adf-ncgr** [LegumeFederation/Workshops#9]
discussed our legume workshop a bit during this intermine call:
https://docs.google.com/document/d/1p2KSzPSKTahzsSUYmWH-ZRM4dcuRlfE67Up6rcVE5bI/edit#

we may be able to leverage some content that the intermine group has already developed for their efforts:
https://www.gen.cam.ac.uk/events/intermine-training

**Comment by vivekkrish** [comment]
@yochannah from InterMine reached out to us and shared an InterMine blog post describing the various training resources available for re-use by members of the community, available here:
https://intermineorg.wordpress.com/2017/11/09/talks-and-workshops-sharing-our-materials-for-re-use/

This would serve as a starting point for material we would prepare for the upcoming LegFed Workshop in March 2018.

**Comment by adf-ncgr** [comment]
@sammyjava do you think it would be worth showing off what you ended up doing at the next intermine outreach meeting? IIRC @jd-campbell is supposed to be part of that one too.

**Comment by sammyjava** [comment]
Nah, I think it's too detailed and specific, I don't think the dev group would care that much. ("Yeah, InterMine works.")
